source/community/about/community-guidelines.html.md

```diff
@@ -8,89 +8,43 @@ wiki_revision_count: 11
 wiki_last_updated: 2015-05-20
 ---
 
-# Community guidelines
+# Community Code of Conduct
 
-We want to keep the oVirt Community a great place to participate, but we need your help to keep it that way. While we have specific guidelines for various tools, in general, you should:
+We want to keep the oVirt Community a great place to participate, but we need your help to keep it that way. The oVirt community is made up of a diverse mix of individuals using and contributing to all aspects of the project from all over the world, and we want to make sure that the community is a safe and friendly place for everyone. 
 
-*   Be nice
-*   Keep it clean
-*   Keep it legal
-*   Stay on topic
+This code of conduct applies equally to founders, mentors and those seeking help and guidance. It applies to all spaces managed by the oVirt project, including IRC, mailing lists, Bugzilla, Gerrit, oVirt events, and any other forums created by the project team which the community uses for communication. 
```

Comment:
I think 'Bugzilla' should not be in the list since bugzilla.redhat.com is not a space managed by the oVirt project.